framework/doc/content/source/physics/DiffusionCG.md

```diff
@@ -0,0 +1,49 @@
+# DiffusionCG
+
+!syntax description /Physics/DiffusionCG/DiffusionCG
+
+See the [DiffusionPhysicsBase.md] documentation for the diffusion equation solved.
+
+## Continuous Galerkin Discretization
+
+The volumetric discretization of the time derivative uses the [TimeDerivative.md] kernel.
+The diffusion term $\nabla \cdot D \nabla u(\vec{x})$ is integrated by parts and represented using:
+- a [Diffusion.md] kernel if no diffusion coefficient is passed
```

Comment:
You need a blank line before each of these lists. Do this in all of your .md files.